### PR TITLE
Split tile creation to use (topo-preserved) simplified geoms at low zooms

### LINKED ISSUE #10: Simplify geometries at lower zooms
Currently, the state/county/district boundaries at low zooms are overly detailed, resulting in slower map load times and a rendered map that is less visually appealing / clean.  I think/hope we should be able to improve this with the right tippecanoe options; if not, we may have to fall back to using mapnik / Studio Classic to render the vector tiles (which would def be less convenient for automating)

### PR DESCRIPTION
**Pull request by anandthakker**
Closes #10

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -938,6 +938,19 @@ geojson/districts.geojson: shp/us/congress-ungrouped.shp
...
+		-o temp.json \
+		--no-pre-quantization \
+		--post-quantization=1e6 \
+		--simplify=7e-6 \
+		--properties GEOID,STATE_FIPS,FIPS\
+		--external-properties fips.csv \
+		-- $^
+	node_modules/.bin/topojson-geojson -o $(dir $@) temp.json
+	mv $(basename $@).json $@
+	rm temp.json
```
Comment (anandthakker) [inline Makefile:952]: @dereklieu ^ this creates a topojson from the full-res boundaries, simplifying the shapes while preserving topology (i.e. shared edges)

File: Makefile [hunk 2]
```diff
@@ -957,6 +970,109 @@ topo/us-%-cities.json: geojson/%/cities.geojson
...
+		| ./flatten-geojson \
+		> $@
+
+election-results/2012-%-10m.geojson: geojson/albers/us-10m/%.geojson
+	cat geojson/albers/us-10m/$*.geojson \
+		| node_modules/.bin/geojson-join \
+			--format=csv --againstField=id --geojsonField=id election-results/2012.csv \
+		| ./add-density-property voteCount \
+		| ./flatten-geojson \
+		> $@
```
Comment (anandthakker) [inline Makefile:1018]: ^ note the duplication here: 2012-%-10m.geojson is analogous to 2012-%.geojson, but derived from the simplified version of the boundary file.

File: Makefile [hunk 3]
```diff
@@ -957,6 +970,109 @@ topo/us-%-cities.json: geojson/%/cities.geojson
...
+		--read-parallel \
+		--no-polygon-splitting \
+		--minimum-zoom=4 \
+		--maximum-zoom=10 \
+		--drop-rate=0 \
+		--name=$*-results \
+		--output $@
+
+tiles/%-results.mbtiles: tiles/%-results.z0-3.mbtiles tiles/%-results.z4-10.mbtiles
+	tile-join -f -o $@ $^
```
Comment (anandthakker) [inline Makefile]: ^ these three targets now make two different tilesets for the different zoom levels, and then merge them together.  tile-join doesn't support inputs w/ overlapping tiles, so important that the tilesets we're merging are disjoint

File: Makefile [hunk 4]
```diff
@@ -957,6 +970,109 @@ topo/us-%-cities.json: geojson/%/cities.geojson
...
+		--read-parallel \
+		--no-polygon-splitting \
+		--minimum-zoom=3 \
+		--maximum-zoom=10 \
+		--drop-rate=0 \
+		--name=$*-results \
+		--output $@
+
+tiles/%-results.mbtiles: tiles/%-results.z0-2.mbtiles tiles/%-results.z3-10.mbtiles
+	tile-join -f -o $@ $^
```
Comment (anandthakker) [inline Makefile:1058]: ^ these three targets now make two different tilesets for the different zoom levels, and then merge them together. tile-join doesn't support inputs w/ overlapping tiles, so important that the tilesets we're merging are disjoint

**Comment by dereklieu** [comment]
These look great live. Merging.

**Pull request merged**